tbx/static_src/javascript/components/desktop-close-menus.js

```diff
@@ -29,6 +29,7 @@ class DesktopCloseMenus {
             this.desktopSubMenus.forEach((item) => {
                 item.closest('[data-has-subnav]').classList.remove('active');
                 item.setAttribute('aria-expanded', 'false');
+                this.body.classList.remove('no-scroll');
```

Comment:
**Suggestion**: When I click outside the menu to close it, an error is raised as `this.body` is `undefined`. I switched it to the code below, which worked, but I don't know if it's acceptable to reference the body like this:

```suggestion
                document.body.classList.remove('no-scroll');
```

Reply:
I'm sorry folks this was a really silly oversight but great catch Ian! I must not have tested correctly. A fix is now pushed 🥳